src/Pim/Bundle/EnrichBundle/Controller/Rest/ProductController.php

```diff
@@ -216,7 +228,8 @@ public function postAction(Request $request, $id)
         $violations->addAll($this->localizedConverter->getViolations());
 
         if (0 === $violations->count()) {
-            $this->productSaver->save($product);
+//            $this->productSaver->save($product);
+            $this->productModelSaver->save($product);
```

Comment:
Other idea: instead of giving `ProductInterface` or `ProductModelInterface` to the `saver` we could give `FlexibleInterface`